### ISSUE TITLE
Reply fallback is included in edited message m.new_content

### ISSUE THREAD

**Issue by tulir** [vector-im/riot-web#11129]
Didn't find an issue for this yet and nobody has done it. Bad behavior was introduced in matrix-org/matrix-react-sdk#3192.

As per the aggregation proposal:
> The edited reply should ditch the fallback representation of the reply itself however from `m.new_content` (specifically the `<mx-reply>` tag in the HTML, and the chevron prefixed text in the plaintext which we don't know whether to parse as we don't know whether this is a reply or not), as we can assume that any client which can handle edits can also display replies natively.

> XXX: make Riot do this

**Comment by t3chguy** [comment]
MSC1849 is still in review in review, so am not 100% whether this can land

**Comment by tulir** [comment]
Why did reactions and edits land then :thinking: 

That quote is from the latest version of MSC1849. Other clients can not safely remove the reply fallback in the edit `m.new_content`, so it must not be included. My comment in the MSC1849 extension (that was already merged) goes into more detail: https://github.com/matrix-org/matrix-doc/pull/2154#discussion_r308357177

**Comment by t3chguy** [comment]
Let me rephrase, *I* know not enough as to whether it can land or not :D
but code-wise it is sane
